Sources/Sentry/SentryProfiler.mm

```diff
@@ -121,6 +126,10 @@
          backtraceAddressIdx < backtrace.addresses.size(); backtraceAddressIdx++) {
         const auto instructionAddress
             = sentry_formatHexAddressUInt64(backtrace.addresses[backtraceAddressIdx]);
+        if (instructionAddress == nil) {
```

Comment:
I don't see a way for the return value of `sentry_formatHexAddressUInt64` to be nil